backend/src/v5/processors/teamspaces/projects/models/commons/tickets.js

```diff
@@ -341,4 +344,33 @@ Tickets.getTicketList = (teamspace, project, model,
 	return getAllTickets(teamspace, project, model, deleteIfUndefined({ projection, updatedSince, sort }));
 };
 
+Tickets.getOpenTicketsCount = async (teamspace, project, model) => {
+	const tickets = await getAllTickets(teamspace, project, model, {
+		projection: { type: 1, [`properties.${basePropertyLabels.STATUS}`]: 1 },
+	});
+
+	let openTicketsCount = 0;
+	const templateToStatuses = { };
+
+	for (let i = 0; i < tickets.length; i++) {
+		const ticket = tickets[i];
+		const templateId = UUIDToString(ticket.type);
+
+		if (!templateToStatuses[templateId]) {
+			// eslint-disable-next-line no-await-in-loop
+			const { config } = await getTemplateById(teamspace, ticket.type, { config: 1 });
```

Comment:
I think instead of this, we may as well get all the templates available within the teamspace in one go.

It's not going to be a lot and we're only getting hte configs out

This way we don't have to interrupt the linter